### DISCUSSION TITLE
HRT4-ZW/SRT321 Thermostat expose ONLY 'Heat' HVAC Mode and Thermostat Setpoint command doesn't work anymore

### DISCUSSION THREAD

**Discussion by Chilgl0rd** [zwave-js/zwave-js#7273] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

HRT4-ZW/SRT321 have multiple thermostat mode (Off, Heat, Cool, Auto...) but ZWave JS only extract Heat mode and not sync the hvac status.
It should be at least Off/Heat
Plus, Action Set Temperature or Action ZWave Set a value doesn't work.
I try with an other type of thermostat (Heatit) and it worked fine
![image](https://github.com/user-attachments/assets/8a0adb8b-2160-425f-a3f0-016d4dec49d3)

### Software versions

zwave-js-ui: 9.21.1
zwave-js: 13.4.0

### Z-Wave Controller (Stick/Dongle/...)

AEOTEC ZWA010-C Gen
FW/SDK 7.21.4 but same issue this previous FW

### Device information

Manufacturer: Secure Meters
Model name: HRT4-ZW / SRT321 
Node ID: 129

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwave_js-612b85679413239739021c48bdda96e4-Climate Control Office-ebc06c3eb3e42406cdcd6d9153e2cbdd.json](https://github.com/user-attachments/files/17343550/zwave_js-612b85679413239739021c48bdda96e4-Climate.Control.Office-ebc06c3eb3e42406cdcd6d9153e2cbdd.json)

[zwave_js-612b85679413239739021c48bdda96e4-Climate Control Office-ebc06c3eb3e42406cdcd6d9153e2cbdd(1).json](https://github.com/user-attachments/files/17343583/zwave_js-612b85679413239739021c48bdda96e4-Climate.Control.Office-ebc06c3eb3e42406cdcd6d9153e2cbdd.1.json)

[a comment by an automated account is not shown]

**Comment by Chilgl0rd** [comment]
[zwave-js-ui-store.zip](https://github.com/user-attachments/files/17343965/zwave-js-ui-store.zip)

**Comment by Chilgl0rd** [comment]
[zwave-js-ui-store(1).zip](https://github.com/user-attachments/files/17345545/zwave-js-ui-store.1.zip)
Hello, I excluded/included it again (ID 130 now), I re-interviewed also. Still a mess :-/

**Comment by AlCalzone** [comment]
The device is reporting only supporting the **Heating** mode. According to [this manual](https://products.z-wavealliance.org/ProductManual/File?folder=&filename=Manuals/1563/SRT321%20User%20MAnual.pdf), this seems to be correct:
![grafik](https://github.com/user-attachments/assets/e94ca94b-e935-4a93-b361-26ad54cf08c5)
